controllers/app/api/v1/actions_types.go

```diff
@@ -20,34 +20,32 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+// action engine type
+const (
+	KubectlAction string = "kubectl"
+)
+
+// action status
+const (
+	Success    string = "SUCCESS"
+	Processing string = "PROCESSING"
+	Failed     string = "FAILED"
+)
 
 // ActionsSpec defines the desired state of Actions
 type ActionsSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
-
-	// Application name, actions will bind to an application
-	AppName string `json:"appName,omitempty"`
-	// Alias will show it on dashboard. like DELETE UPDATE MIGRATE...
-	Alias string `json:"alias,omitempty"`
-	// Args, the actions input args, like oss backup address...
-	Args map[string]string `json:"args,omitempty"`
-	// Actions exec command, support render some args and downward API, like application name, service name ...
-	CMD string `json:"cmd,omitempty"`
+	AppName    string            `json:"appName,omitempty"`
+	ActionName string            `json:"actionName,omitempty"`
```

Comment:
What is the difference with meta.Name？

Reply:
eeee,that is your code ,  I forgot to move it . I'll fix it

Reply:
no difference